### ISSUE TITLE
Access to ChEBI patent corpus through http

### ISSUE THREAD

**Issue by yetinam** [ebi-chebi/ChEBI#3771]
Dear developers,
we are currently working on integrating a large set of biomedical NER corpora into the FLAIR NLP framework (https://github.com/flairNLP/flair/issues/1513) and would like to include the ChEBI patent corpus. Therefore I wanted to ask, if the ChEBI corpus is accessible somewhere through http? So far I only found the CVS version (http://chebi.cvs.sourceforge.net/), but we would prefer not to have a dependency on CVS.
If it is currently not available over http, could you make it available?

**Comment by muthuvenkat** [comment]
@yetinam,  ChEBI links to patent data and also resources like SureChembl (chemicals to patent). Please refer the [downloads](https://www.ebi.ac.uk/chebi/downloadsForward.do) and the [documentation](https://docs.google.com/document/d/11G6SmTtQRQYFT7l9h5K0faUHiAaekcLeOweMOOTIpME/).Hope this helps !

**Comment by yetinam** [comment]
Hey @muthuvenkat ,
thanks for the quick reply. I'm looking specifically for the ChEBI corpus for named entity recognition, that contains full texts with annotated entities, as in the chapati/patentsGoldStandard subfolder of the CVS (http://chebi.cvs.sourceforge.net/). This seems not to be part of the ChEBI database itself.
I rechecked both the downloads section and the documentation you referenced, but I do not find any mention of the annotated full texts there anymore. I also did not find a reference in the ChEBI user manual. Are the annotated full texts still provided on the current ChEBI page or only on the legacy version?
